main.tex

```diff
@@ -221,71 +221,73 @@
+\titleSection{On the adversarial examples problem}\label{On the adversarial examples problem}
 
-\titleSection{Cyber protocol}\label{cyber protocol}
+\linkgreen{https://ipfs.io/ipfs/QmeS4LjoL1iMNRGuyYSx78RAtubTT2bioSGnsvoaupcHR6}{The current architecture of search engines} is a means in which one entity processes and then ranks all the 'shit'. This approach suffers from one difficult but a distinct problem, that has yet to be solved even by the brilliant Google scientists: \linkgreen{https://ipfs.io/ipfs/QmNrAFz34SLqkzhSg4wAYYJeokfJU5hBEpkT4hPRi226y9}{the adversarial example problem}. The problem that Google acknowledges is that it is rather difficult to algorithmically reason, whether or not, a particular sample is adversarial. That is - independently of how 'neat' a learning technology in itself is. A crypto-economical approach can change beneficiaries within the game. Consequently, this approach will effectively remove possible Sybil-attack vectors; removes the necessity to make a decision on example crawling, and the significance of extracting results from one entity to the whole world. Learning Sybil-resistant models will probably lead to orders of magnitude of more predictive results.
 
-In it's core the protocol is minimalistic and can be expresses in the following steps:
+\titleSection{Cyber protocol}\label{Cyber protocol}
+
+In its core the protocol is very minimalistic, and can be expressed via the following steps:
 
 \begin{enumerate}[nosep]
-\item Compute genesis of cyber protocol based on the distribution rules defined by this paper
-\item Define knowledge graph state
-\item Take cyberlinks
-\item Check the validity of signatures
-\item Check the validity of CIDv1
-\item Check the bandwidth limit
-\item If signatures, bandwidth limit, and CIDv1 are valid then apply cyberlinks
-\item Every round calculate cyber•rank deltas for the knowledge graph
+\item Computing the genesis of cyber-protocol, based on the distribution rules defined in this paper;
```

Comment:
Please remove ; out of the whole document. I really don't like it, hence didn't use.

Reply:
Done